lib/contentful.ts

```diff
@@ -109,6 +109,13 @@ export interface CreateClientParams {
    * Interceptor called on every response. Takes Axios response object as an arg.
    */
   responseLogger?: (response: AxiosResponse<any> | Error) => unknown
+
+  /**
+   * Enable alpha features.
+   */
+  alphaFeatures?: {
+    withContentSourceMaps?: boolean
```

Comment:
It would be helpful If you add a JS Doc here explaining briefly what is this about for customers. It will show up to the developer as a help text in the editor completion menu.